client/js/components/GameView/GameMenu.vue

```diff
@@ -57,19 +60,70 @@ export default {
     return {
       showGameMenu: false,
       showConcedeDialog: false,
-      conceding: false,
+      showStalemateDialog: false,
+      loading: false,
     };
   },
+  computed: {
+    showDialog: {
+      get() {
+        return this.showConcedeDialog || this.showStalemateDialog;
+      },
+      set(newVal) {
+        if (!newVal) {
+          this.showConcedeDialog = false;
+          this.showStalemateDialog = false;
+        }
+        this.showGameMenu = false;
+      },
+    },
+    dialogTitle() {
+      return this.showConcedeDialog ? 'Concede' : 'Request Stalemate';
+    },
+    dialogText() {
+      return this.showConcedeDialog
+        ? 'The game will end and your opponent will win.'
+        : 'If your opponent agrees, the game will end in a stalemate. The request will cancel if the opponent declines or either player makes a move';
+    },
```

Comment:
We should eventually move all of these strings to a localization json file for ease of translating in the future.

Reply:
agreed